src/registrar/admin.py

```diff
@@ -2864,23 +2864,196 @@ def save_model(self, request, obj, form, change):
 
 
 class PortfolioAdmin(ListHeaderAdmin):
-
     change_form_template = "django/admin/portfolio_change_form.html"
+    fieldsets = [
+        # created_on is the created_at field, and portfolio_type is f"{organization_type} - {federal_type}"
+        (None, {"fields": ["portfolio_type", "organization_name", "creator", "created_on", "notes"]}),
+        # TODO - uncomment in #2521
+        # ("Portfolio members", {
+        #     "classes": ("collapse", "closed"),
+        #     "fields": ["administrators", "members"]}
+        # ),
+        ("Portfolio domains", {"classes": ("collapse", "closed"), "fields": ["domains", "domain_requests"]}),
```

Comment:
Worth asking now whether design would accept collapse or prefer collapse--dgfieldset (domain request pattern). If latter, Domains would have to come after Domain Requests for that pattern to work here.